compiler/compile/OMRCompilation.cpp

```diff
@@ -2079,6 +2085,44 @@ void OMR::Compilation::verifyCFG(TR::ResolvedMethodSymbol *methodSymbol)
       }
    }
 
+
+/*
+ * \brief
+ *    Check to make sure the rdbars are anchored and anchor a rdbar if it's found unanchored
+ *
+ * \note
+ *    Ideally all optimizations should anchor a rdbar when it's created.
+ *    This API is for fixing the unanchored rdbars in case some optimizations forget to
+ *    anchor the node by mistake. Optimizations should still try to generate correct rdbar
+ *    trees instead of relying on this API.
+ */
+void OMR::Compilation::verifyAndFixRdbarAnchors()
```

Comment:
Please prefer to add API Doxygen information in the header file such that consumers looking at the API interface (the header) can understand what a function does. Users may not always have access to the source code of the implementation and they should not need it to understand API semantics.

Reply:
moved Doxygen comment into header file. Thanks!